### ISSUE TITLE
can't drain life

### ISSUE THREAD

**Issue by Willowflame** [Exzotic5485/Reaper-Origin#2]
when I sneak right click to drain life with the diamond or netherite scythe it just teleports me to limbo unless the ability is on cooldown.

**Comment by Exzotic5485** [comment]
Yeah, ill make it so that doesnt happen while sneaking but for now you can use life drain without holding a scythe.
